projects/storefrontlib/src/cms-components/checkout/components/payment-method/billing-address-form/billing-address-form.component.html

```diff
@@ -90,6 +90,49 @@
         />
       </label>
     </div>
+    <div class="form-group col-md-6">
```

Comment:
I checked the POST request of "payment/sop/response", and I found that the "billTo_State" is the same as "shipTo_State", even I select a different state.